ambry-quota/src/main/java/com/github/ambry/quota/AmbryQuotaManager.java

```diff
@@ -147,6 +153,19 @@ public QuotaConfig getQuotaConfig() {
     return quotaConfig;
   }
 
+  /**
+   * Notify {@link QuotaSource}s about creation of new Ambry {@link Account} or {@link com.github.ambry.account.Container}.
+   * @param updatedAccounts {@link Collection} of {@link Account}s updated.
+   */
+  protected void onAccountUpdateNotification(Collection<Account> updatedAccounts) {
```

Comment:
`accountUpdateConsumer` callback will be invoked not only when the new account/container is created, but also when the existing account/container is updated, like encryption flag, cache flag, etc..
If we only want `QuotaSource` to deal with newly created account/container, we should probably keep a list of account/container in `AmbryQuotaManager` so we can filter the updates out.

Reply:
Then please update the comments for this method.

Reply:
done.